### PR TITLE
2.x: Elide more create/callback wrappers in favor of direct Observables.

### PR DESCRIPTION
**Pull request by JakeWharton**

### CODE REVIEW

File: src/main/java/io/reactivex/Observable.java [hunk 1]
```diff
@@ -450,35 +450,35 @@ public Throwable call() {
 
     public static <T> Observable<T> fromPublisher(final Publisher<? extends T> publisher) {
         Objects.requireNonNull(publisher, "publisher is null");
-        return create(new ObservableConsumable<T>() {
+        return new Observable<T>() {
```
Comment (akarnokd) [inline src/main/java/io/reactivex/Observable.java:453]: There should be another run to change such anonymous classes to proper internal classes.

**Comment by akarnokd** [comment]
:+1:

File: src/main/java/io/reactivex/Observable.java (hunk 1 above)
Comment (JakeWharton) [inline src/main/java/io/reactivex/Observable.java:453]: I'll add it to my queue.

**Comment by akarnokd** [comment]
:+1:

**Pull request merged**